.github/workflows/dockerbuild.yml

```diff
@@ -39,8 +39,9 @@ jobs:
     - name: Build Images
       env: 
         # Every folder in the repo that has a Dockerfile within it, comma separated 
-        DOCKERFOLDERS: "edge-detect/ai,tracker,piaware,pan-tilt-pi,notebook-server,mqtt,egi,axis-ptz"
+        DOCKERFOLDERS: "edge-detect/coral,edge-detect/yolo,axis-ptz,egi,mqtt,notebook-server,piaware,tracker"
         PROJECT_NAME: "skyscan"
+        REPO_NAME: "meadejs"
```

Comment:
I think this needs to be changed to the correct name

Reply:
Addressed in https://github.com/IQTLabs/edgetech-coral-detect/pull/1